scripts/dts/extract/flash.py

```diff
@@ -10,6 +10,11 @@
 from extract.default import default
 from extract.reg import reg
 
+flash_dev_label_prefix = {
+    "soc-nv-flash":"soc_flash",
+    "serial-flash":"spi_flash"
```

Comment:
can you change to "serial-flash":"serial_flash" or another equivalent?

Reply:
This array is for compatibility with old code
The old code uses the name spi flash

Reply:
Modified to "serial-flash": "serial_flash" and directly delete "serial-flash": "spi_flish" will have the same effect

Reply:
What a old code? Can be modified as well?

> Modified to "serial-flash": "serial_flash" and directly delete "serial-flash": "spi_flish" will have the same effect

I don't understand what you want to say.

Reply:
These two names were defined and used in the previous code.
If there are other places where these two definitions are used, I am in order to be compatible with the previous name.
Amount, this period of time comes in contact, the current zephyr code does not have to consider the name compatibility.
This paragraph can be deleted.